providers/src/airflow/providers/edge/executors/edge_executor.py

```diff
@@ -129,6 +129,34 @@ def execute_async(
             )
         )
 
+    @provide_session
+    def queue_workload(
+        self,
+        workload: Any,  # Note actually "airflow.executors.workloads.All" but not existing in Airflow 2.10
+        session: Session = NEW_SESSION,
+    ) -> None:
+        """Put new workload to queue. Airflow 3 entry point to execute a task."""
+        from airflow.executors import workloads
+
+        if not isinstance(workload, workloads.ExecuteTask):
+            raise TypeError(f"Don't know how to queue workload of type {type(workload).__name__}")
+
+        task_instance = workload.ti
+        key = task_instance.key
+        session.add(
+            EdgeJobModel(
```

Comment:
If this is referring to a TI it should also have the  UUID column from `task_instance.id`

Reply:
I need to switch the model - but rather in a later PR. As atm this is not critical and Airflow 2 does not have the UUID. But once we can drop Airflow 2 support that totally makes sense.